packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -701,6 +736,9 @@ export const useArbTokenBridge = (
     const tx = await erc20Bridger.withdraw({
       l2Signer,
       erc20l1Address: erc20L1Address,
+      // TODO: Isn't L1 address the same as destination address? Both params are required.
+      // What's the difference between the two?
+      destinationAddress: erc20L1Address,
```

Comment:
Is `destinationAddress` where you want the tokens to land on L1, @yahgwai? If that's case, just use `walletAddress`, no need to add an additional parameter to the function.